Direct users to `cargo concordium test --out ..` instead `cargo test` (#837)
Concordium/concordium.github.io: source/mainnet/smart-contracts/tutorials/piggy-bank/testing.rst

This produces the Wasm module ``piggy_bank_part2.wasm.v1`` in the root of your project, unless you have any typos or bugs in your code.
If that is the case, try to fix them using the helpful error messages from the compiler or go back to the end of part 1 and copy the full contract code again.

.. note::

   Since the tests run against the compiled Wasm module, there is a risk of accidentally using an outdated Wasm module.

>    To circumvent this, cargo concordium's test command can both *build* and test you contract, which ensures that you always test the newest version of your code.

Reviewer: ```suggestion
   To circumvent this, cargo concordium's test command can both build *and* test your contract, which ensures that you always test the newest version of your code.
```

>    To circumvent this, cargo concordium's test command can both *build* and test you contract, which ensures that you always test the newest version of your code.

Reviewer 2: Perhaps we could add here that if you use `cargo concordium test --out` then you 
- build the module and output it
- and run tests on it.

And thus you should deploy that module, which means you are deploying a module you have tested, exactly that one.

Reviewer 2: Yeah, I think that's good.

>    To do so, use the ``--out`` parameter testing:

Reviewer: ```suggestion
   To do so, use the ``--out`` parameter testing:
```
```suggestion
   To do so, use the ``--out`` parameter when testing:
```

.. code-block:: console

   $cargo concordium test --out piggy_bank_part2.wasm.v1

> Note that we are using the ``--out`` flag for ``cargo concordium`` to ensure that the module is freshly compiled and thus matches the newest version of your smart contract code.

Reviewer: ```suggestion
Note that the ``--out`` flag for ``cargo concordium`` ensures that the module is freshly compiled and thus matches the newest version of your smart contract code.
```

> Note that we are using the ``--out`` flag for ``cargo concordium`` to ensure that the module is freshly compiled and thus matches the newest version of your smart contract code.

Author: Right, I've tried to rephrase it to make it more clear.
